### DISCUSSION TITLE
Policy-Recommendation?

### DISCUSSION THREAD

**Discussion by queen4me** [trapexit/mergerfs#1291] [General]
I must admit that I'm absolutely lost with the multitude of different policies for mergerfs and the advantages and disadvantages.
Could you please give me a recommendation for my usecase?
The mergerfs-pool is used to store my multimedia collection so the following applies:
-don't want to care about how much space ist available on which drive
-just create new directories and copy files
-directories should not be "spreaded" over drives to avoid accessing all drives in worst case

thanks a lot for your recommendations/help.

**Comment by trapexit** [comment]
> -just create new directories and copy files

I don't know what this means.

> -directories should not be "spreaded" over drives to avoid accessing all drives in worst case

I don't know what this means either. What do you mean to "avoid accessing all drives"? Naturally mergerfs has to access all branches for most functions.

**Reply by tripleo1** [reply]
>> -directories should not be "spreaded" over drives to avoid accessing all drives in worst case

> I don't know what this means either. What do you mean to "avoid accessing all drives"? Naturally mergerfs has to access all branches for most functions.

$ ls  mergerfs/random

-- wake up all drives

vs

$ ls mergerfs/constrained-dir

-- wake up only one drive

**Reply by trapexit** [reply]
That's not a thing.

https://github.com/trapexit/mergerfs/wiki/Limit-Drive-Spinup

**Reply by tripleo1** [reply]
"Not my question", but this is the answer.

thx

**Reply by queen4me** [reply]
Exact. Thanks a lot trapexit and tripleo1.
